### ISSUE TITLE
re-write in layman terms

### ISSUE THREAD

**Issue by GA-CyberWorkforceAcademy** [gacybercenter/kinetic-docs#7]
Specify how this is capable and the process in a layman manner (address non-traditional platforms and define and specify capability/process for HWIL):

<The GCR leverages QEMU and KVM acceleration (where applicable/available) to virtualize all kinds of traditional and non-traditional platforms, all the way from individual binaries to entire systems. If for some reason your platform cannot be virtualized, the GCR fully supports Hardware-in-the-Loop (HWIL) and connecting arbtirary devices to your GCR project spaces is fast and easy.>

**Comment by gacyberrange** [comment]
transferred upstream: https://gitlab.com/gacybercenter/gacyberrange/infrastructure/kinetic-docs/-/issues
